lib/assisted_workflow/addons/pivotal.rb

```diff
@@ -46,14 +45,24 @@ def pending_stories(options = {})
       log "loading pending stories"
       states = ["unstarted"]
       states << "started" if options[:include_started]
-      @project.stories.all(:state => states, :owned_by => @fullname, :limit => 5)
+      filter_str = "state:#{states.join(',')} owned_by:#{@client.me.id}"
+      @project.stories(:filter => filter_str, :limit => 5)
     end
     
     def valid?
       !@project.nil?
     end
-  
+
     private
+
+    def add_comment_to_story(story, text)
+      url = "/projects/#{story.project_id}/stories/#{story.id}/comments"
+      @client.post(url, params: {:text => text})
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.